.gitignore

```diff
@@ -7,7 +7,7 @@
 /phoebe/atmospheres/tables/phoenix*
 /phoebe/atmospheres/tables/tmap*
 /phoebe/atmospheres/tables/tlusty
-/phoebe/atmospheres/tables/phoenix.orig
+/tests/tests/test_passbands/tables
```

Comment:
nothing seems to be written to this directory by the code, so I'm not sure it makes sense to add to the gitignore

Reply:
that's right, but this is a symbolic link to the (extensive) model atmosphere tables that are needed for the passband test. The logic there is to see whether tables are present and, if so, to run the test, otherwise skip it. Either way, git should not try to index those files, hence the entry in .gitignore.